perun-base/src/main/resources/perun-roles.yml

```
@@ -6207,6 +6207,40 @@ perun_policies:
     include_policies:
       - default_policy
 
+  getAllBlockedLoginsInNamespaces_policy:
+    policy_roles:
+      - PERUNOBSERVER:
+    include_policies:
+      - default_policy
+
+  isLoginBlocked_policy:
```

I think in the name of conventions, we should add type of method parameters into policy names.
```suggestion
  isLoginBlocked_String_policy:
```